<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddInvoice</class>
 <widget class="QDialog" name="DialogAddInvoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>675</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>manageBill Add Invoice</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/image/receipt16.png</normaloff>:/icon/image/receipt16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelTitle">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;URW Palladio L&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Add Invoice &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelSelect">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Select :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Search</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/check24.png</normaloff>:/icon/image/check24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelOr">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>or</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButtonAddHuman">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/add24.png</normaloff>:/icon/image/add24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
</string>
       </property>
       <property name="text">
        <string>Company / Family Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
</string>
       </property>
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelBillingDate">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Billing Date :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditBillingDate"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelPaymentDate">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Payment date :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEditPaymentDate"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="labelTotalAmount">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Total Amount : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTotalAmount"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelReason">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Reason :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEditReason"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxBillPaid">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Bill Paid</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonAddProduct">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Add Product</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/add24.png</normaloff>:/icon/image/add24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="labelProduct">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Products :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QTableWidget" name="tableWidgetProduct">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>220</height>
      </size>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;URW Palladio L&quot;;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>125</number>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>VAT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>net Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type of quantity</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="9" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonAddPDF">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;URW Palladio L&quot;;
</string>
       </property>
       <property name="text">
        <string>&amp;Add PDF</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/add24.png</normaloff>:/icon/image/add24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TEST&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/add24.png</normaloff>:/icon/image/add24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonBack">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Back</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/undo24.png</normaloff>:/icon/image/undo24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
